docs/pass-through.md

```diff
@@ -0,0 +1,237 @@
+# Running Antrea In Pass-Through Mode
+
+Antrea supports chaining with other CNI implementations such as EKS, AKS. In this mode, 
+Antrea enforces k8s network policies, and delegates Pod IP management
```

Comment:
k8s -> Kubernetes or K8s (the former is better if this is assumed to be user documentation).
NetworkPolicies - capitalize the first letter.

Reply:
will do